awscli/customizations/s3/results.py

```diff
@@ -239,35 +246,40 @@ class ResultPrinter(BaseResultHandler):
         '{transfer_type} failed: {src} to {dest} {exception}'
     )
     WARNING_FORMAT = (
-        'warning: {message}'
+        '{message}'
```

Comment:
I would actually prefer that. If I had the option I would:

1) Remove the prefix "warning: " from ``create_warning``
2) Add "warning: " to the format string

The only issue is that file generators still use ``create_warning`` and those warning still get delegated to the old stuff. How would a TODO right here sound?

Then for "error: " the only reason I did not do it was because it would deviate from existing behavior in the CLI. We do not prepend "error: " to these type of errors. I would be fine in adding it though if it makes sense?